flocker/acceptance/endtoend/test_dockerplugin.py

```diff
@@ -54,8 +45,29 @@ def get_path(name):
             # do verify certificate authority signed the server certificate:
             assert_hostname=False,
             verify=get_path(b"cluster.crt"))
-        client = Client(base_url="https://{}:{}".format(address, DOCKER_PORT),
-                        tls=tls, timeout=100)
+        return Client(base_url="https://{}:{}".format(address, DOCKER_PORT),
+                      tls=tls, timeout=100)
+
+    def run_python_container(self, cluster, address, docker_arguments, script,
+                             script_arguments, cleanup=True):
+        """
+        Run a Python script as a Docker container with the Flocker volume
+        driver.
+
+        This is a blocking call.
+
+        :param Cluster cluster: Description of the cluster we're talking to.
+        :param bytes address: The public IP of the node where it will run.
+        :param dict docker_arguments: Additional arguments to pass to
+            Docker run call.
```

Comment:
:question: This docstring is a bit misleading since we're not calling the `docker run` command but passing `HostConfig` dict to the Docker API.